projects/js-toolkit/packages/portal-base/src/start/customElement.ts

```diff
@@ -37,4 +54,108 @@ export default async function customElement(project: Project): Promise<void> {
 	const server = new webpackDevServer(devServerOptions, compiler);
 
 	await server.start();
+
+	process.on('exit', () => {
+		print(warn`
+Undeployed live development Remote App from Liferay.`);
+		print(info`
+Run 'yarn deploy' if you want to make your Remote App available in your Liferay
+server again by deploying a non-live version of it.`);
+
+		fs.rmSync(zipFile.asNative);
+	});
+}
+
+async function checkForOldZipFile(zipFile: FilePath): Promise<void> {
+	if (!fs.existsSync(zipFile.asNative)) {
+		return;
+	}
+
+	const stats = fs.statSync(zipFile.asNative);
+
+	if (Date.now() - stats.mtimeMs > 1000 * 60 * 120) {
+		const lines = createInterface({
+			input: process.stdin,
+		});
+
+		print(warn`
+There is an older than 2 hours version of this project in your Liferay server.`);
```

Comment:
Why the 2 hour limit? Was this just arbitrary?

Reply:
Yes. It seems reasonable to me. My intention is to warn only in cases where you may have deployed the application some time ago, in a previous development session, but not every time you run `yarn start`. Two hours seems enough to not trigger false positives when you stop for lunch :sweat_smile: 

However, I'm now thinking that people won't be running `yarn start` a lot of times. Chances are that they just run it once per development session, so maybe we can warn always (without checking for the elapsed time)... :thinking:

Reply:
Sounds good, just wanted to make sure I fully understood!

Reply:
I'm going to remove the check for the two hours...